eras/babbage/test-suite/src/Test/Cardano/Ledger/Babbage/Serialisation/Generators.hs

```diff
@@ -68,16 +68,16 @@ instance
       <$> arbitrary
       <*> arbitrary
       <*> arbitrary
-      <*> arbitrary
-      <*> arbitrary
+      <*> scale (`div` 15) arbitrary
+      <*> scale (`div` 15) arbitrary
```

Comment:
You already adjusted the `TxOut`, so no need to scale it again. This effectively would resize the value to an empty map all the time in the TxOut
```suggestion
      <*> arbitrary
      <*> arbitrary
```